src/Application/Components/ClickCanvas/ClickCanvas.jsx

```diff
@@ -34,6 +34,8 @@ const ClickCanvas = React.createClass({
         left: this.imgPosX,
         top: this.imgPosY,
         selectable: false,
+        scaleX: 0.20,
+        scaleY: 0.20,
```

Comment:
Since this is temporary, it's probably time to start making real pictures...or getting some art student to do it.